app/api/geocoding/reverse/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function GET(request: NextRequest) {
+  try {
+    const { searchParams } = new URL(request.url);
+    const lat = searchParams.get('lat');
+    const lon = searchParams.get('lon');
+
+    const response = await fetch(
+      `https://api-test.openepi.io/geocoding/reverse?lat=${lat}&lon=${lon}`
+    );
+    const product = await response.json();
+
+    return NextResponse.json({
+      message: 'Location retrieved successfully',
+      status: 200,
```

Comment:
the HTTP status code shouldn't be in the JSON response body, it should be specified like you do on line 23 I'm not sure if we need the `message`-column there either, but you can make that call.